src/game.cpp

```diff
@@ -8213,12 +8215,84 @@ tripoint game::look_debug()
 ////////////////////////////////////////////////////////////////////////////////////////////
 
 void game::print_all_tile_info( const tripoint &lp, WINDOW *w_look, int column, int &line,
-                                bool mouse_hover )
+                                const int last_line, visibility_type visibility )
+{
+    switch( visibility ) {
+        case VIS_CLEAR:
+            print_terrain_info( lp, w_look, column, line );
+            print_fields_info( lp, w_look, column, line );
+            print_trap_info( lp, w_look, column, line );
+            print_object_info( lp, w_look, column, line, last_line );
+            break;
+        case VIS_BOOMER:
+        case VIS_BOOMER_DARK:
+        case VIS_DARK:
+        case VIS_LIT:
+        case VIS_HIDDEN:
+            print_visibility_info( w_look, column, line, visibility );
+            break;
+        default:
```

Comment:
Ahh, I've seen comiler warnings for this in other languages, but didn't realize there was a compiler warning for it in gcc, hence making it as loud as reasonably possible when an enum was _not_ handled by the switch. :)